src/main/java/cubicchunks/util/CubeCoords.java

```diff
@@ -31,16 +31,11 @@
 import java.util.function.Consumer;
 import java.util.function.Predicate;
 
-import static cubicchunks.util.Coords.CUBE_MAX_X;
-import static cubicchunks.util.Coords.CUBE_MAX_Y;
-import static cubicchunks.util.Coords.CUBE_MAX_Z;
-import static cubicchunks.util.Coords.HALF_CUBE_MAX_X;
-import static cubicchunks.util.Coords.HALF_CUBE_MAX_Y;
-import static cubicchunks.util.Coords.HALF_CUBE_MAX_Z;
 import static cubicchunks.util.Coords.blockToCube;
 import static cubicchunks.util.Coords.getCubeXForEntity;
 import static cubicchunks.util.Coords.getCubeYForEntity;
 import static cubicchunks.util.Coords.getCubeZForEntity;
+import static cubicchunks.world.cube.Cube.SIZE;
```

Comment:
SIZE itself does not have any fixed meaning with respect to CubeCoords and Cube.SIZE isn't much longer.